lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -13,7 +13,7 @@ module Resources
         },
         'text': {
           'status': 'generated',
-          'div': '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Appointment</b></p><p><b>Status</b>: booked</p>'\
+          'div': '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Appointment</b></p><p><b>Status</b>: Booked</p>'\
                  '<p><b>Service Type</b>: Established Patient</p><p><b>Start</b>: Jul  8, 2020  1:00 P.M. UTC</p>'\
```

Comment:
Do these need to be update to C1941 examples or are they already from that domain?

Reply:
These examples are from C1941 domain.